### DISCUSSION TITLE
Not able to come back to the application after executing an Ideal Payment

### DISCUSSION THREAD

**Discussion by joseAlbertoVigo** [flutter-stripe/flutter_stripe#250] [Q&A]
This problem is related with the finalization of the payments when you a redirect outside of the app (in my case iDEAL payments). In my case, the app is jumping correctly to bank web page or bank app. There the user can execute the payment successfully. Once the users finalized the payment, they need to be redirect to the app again where they can continue shopping or seeing the status of their order. 
 My problem lies with the fact that after executing the payment, I am not able to come back to the app. The flow is stopped at the Safari screen with a button saying: "Return to the Merchant". When I pressed that button, there is no action there and the only way to come back is pressing the Close button.

Please advise me what I am missing in the configuration or in the code to be able to come back to the app.

**Comment by NeatFastro** [comment]
How about android, is redirecting to the app fine? For me it is but my only problem is the the url extension .de instead of .nl
Is there any way i can chan change it?

**Comment by NeatFastro** [comment]
Also have you added the necessary url schemes and info.plist code into your flutter project?
